code/datums/progressbar.dm

```diff
@@ -59,9 +63,14 @@
 	if(!bars.len)
 		LAZYREMOVE(user.progressbars, bar.loc)
 
-	if (client)
-		client.images -= bar
-	qdel(bar)
+	animate(bar, alpha = 0, time = 5)
+	addtimer(CALLBACK(src, /datum/progressbar/proc/remove_from_client), 5, TIMER_CLIENT_TIME)
```

Comment:
shouldn't this be `addtimer(CALLBACK(src, .proc/remove_from_client), 5, TIMER_CLIENT_TIME)`?